internal/metrics/metrics.go

```diff
@@ -89,6 +91,13 @@ const (
 // to regenerate the metrics documentation.
 func NewMetrics(registry *prometheus.Registry) *Metrics {
 	m := Metrics{
+		buildInfoGuage: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: BuildInfoGuage,
+				Help: "Build information for Contour",
+			},
+			[]string{"version"},
```

Comment:
Check the blog post that @youngnick referenced in the issue. We should expose 'branch', 'goversion', 'revision', and 'version' (arguably goversion isn't useful, but it's best to be predictable and conventional with metrics).

Reply:
Sure thing. Go version seems to already be instrumented through the standard go lib.

```
/ # curl -s contour.projectcontour:8000/metrics | grep go_info
# HELP go_info Information about the Go environment.
# TYPE go_info gauge
go_info{version="go1.13.8"} 1
```
I will experiment with git commands to get the desired labels.